client/src/app/+my-account/my-account-history/my-account-history.component.html

```diff
@@ -1,7 +1,7 @@
 <div class="top-buttons">
   <div class="history-switch">
     <p-inputSwitch [(ngModel)]="videosHistoryEnabled" (ngModelChange)="onVideosHistoryChange()"></p-inputSwitch>
-    <label i18n>Enable video history</label>
+    <span class="ml-2 text-uppercase" i18n>Video history</span>
```

Comment:
I think we should keep a verb for checkboxes (enable seems okay in this case)

Reply:
I don't think the "Enable" adds information.